src/frontend/context/OppgaverContext.tsx

```diff
@@ -0,0 +1,108 @@
+import { AxiosError } from 'axios';
+import createUseContext from 'constate';
+import React from 'react';
+
+import { IOppgave, SaksbehandlerFilter, IDataForManuellJournalføring } from '../typer/oppgave';
+import { byggFeiletRessurs, byggTomRessurs, Ressurs, RessursStatus } from '../typer/ressurs';
+import { useApp } from './AppContext';
+
+const [OppgaverProvider, useOppgaver] = createUseContext(() => {
+    const [dataForManuellJournalføring, settDataForManuellJournalføring] = React.useState(
+        byggTomRessurs<IDataForManuellJournalføring>()
+    );
+    const [oppgaver, settOppgaver] = React.useState(byggTomRessurs<IOppgave[]>());
+    const { axiosRequest } = useApp();
+
+    const hentDataForManuellJournalføring = (oppgaveId: string) => {
+        axiosRequest<IDataForManuellJournalføring, void>({
+            method: 'GET',
+            url: `/familie-ba-sak/api/oppgave/${oppgaveId}`,
+        })
+            .then((hentetDataForManuellJournalføring: Ressurs<IDataForManuellJournalføring>) => {
+                settDataForManuellJournalføring(hentetDataForManuellJournalføring);
+            })
+            .catch((error: AxiosError) => {
+                settOppgaver(byggFeiletRessurs('Ukjent feil ved henting av oppgave', error));
+            });
+    };
+
+    const hentOppgaver = (
+        behandlingstema?: string,
+        oppgavetype?: string,
+        enhet?: string,
```

Comment:
If it is planned for more filtering options here - should this be an object? 

Then you can do e.g.
```
new URLSearchParams({...}).toString
```
to get the string of the params.

Reply:
Interesting! I do not know this API before. Thank you and I will refactor as you suggest👍